api/app/db/crud/hfi_calc.py

```diff
@@ -79,6 +80,22 @@ def store_hfi_request(session: Session, hfi_result_request: HFIResultRequest, us
     session.add(hfi_request)
 
 
+def store_hfi_station(session: Session, station_code: int, fuel_type_id: int, planning_area_id: int, order: int):
+    """ Store planning weather station """
+    planning_weather_station = PlanningWeatherStation(station_code=station_code,
+                                                      fuel_type_id=fuel_type_id,
+                                                      planning_area_id=planning_area_id,
+                                                      order_of_appearance_in_planning_area_list=order)
+    session.add(planning_weather_station)
+
+
+def get_last_station_in_planning_area(session: Session, planning_area_id: int) -> PlanningWeatherStation:
```

Comment:
Shouldn't sonarcloud be complaining about the missing documentation comment for this function?...